src/views/maintenance/maintenance-router.ts

```diff
@@ -0,0 +1,17 @@
+import Maintenance from "@/views/maintenance/index.vue";
+import MaintenanceHome from "@/views/maintenance/home.vue";
+
+export const maintenanceRouter = {
+  path: "/maintenance",
+  component: Maintenance,
+  children: [
+    {
+      path: "",
+      component: MaintenanceHome
+    },
+    {
+      path: "*",
+      redirect: "/maintenance"
+    }
```

Comment:
what is * path?

Reply:
just like regex, it matches anything that is not matched all of above